setup.py

```diff
@@ -4,85 +4,54 @@
 import sys
 import site
 from setuptools import setup, find_packages
+import importlib.util
 
 # Enables --editable install with --user
 # https://github.com/pypa/pip/issues/7953
 site.ENABLE_USER_SITE = "--user" in sys.argv[1:]
 
-NAME = "becquerel"
-
-MAJOR = 0
-MINOR = 3
-MICRO = 0
-VERSION = f"{MAJOR}.{MINOR}.{MICRO}"
-
-DESCRIPTION = __doc__.split("\n")[0].split(": ")[-1]
-URL = "https://github.com/lbl-anp/becquerel"
-MAINTAINER = "The Becquerel Development Team"
-EMAIL = "becquerel-dev@lbl.gov"
+_spec = importlib.util.spec_from_file_location(
+    "__metadata__", "./becquerel/__metadata__.py"
+)
+METADATA = importlib.util.module_from_spec(_spec)
+_spec.loader.exec_module(METADATA)
 
-# classifiers from list at https://pypi.org/classifiers/
-CLASSIFIERS = """\
-Development Status :: 3 - Alpha
-Intended Audience :: Science/Research
-License :: Other/Proprietary License
-Operating System :: OS Independent
-Programming Language :: Python
-Programming Language :: Python :: 3
-Programming Language :: Python :: 3 :: Only
-Programming Language :: Python :: 3.6
-Programming Language :: Python :: 3.7
-Programming Language :: Python :: 3.8
-Topic :: Scientific/Engineering
-Topic :: Scientific/Engineering :: Physics
-"""
+# Enables --editable install with --user
+# https://github.com/pypa/pip/issues/7953
+site.ENABLE_USER_SITE = "--user" in sys.argv[1:]
 
-with open("README.md", "r") as fh:
-    README = fh.read()
 # remove package title from description
-README = "\n".join(README.split("\n")[2:])
+with open("README.md", "r") as fh:
+    README = "\n".join(fh.readlines()[2:])
 
 with open("CONTRIBUTING.md", "r") as fh:
     CONTRIBUTING = fh.read()
 
-with open("LICENSE.txt", "r") as fh:
-    LICENSE = fh.read()
-
 with open("requirements.txt", "r") as fh:
-    REQUIREMENTS = fh.read()
+    REQUIREMENTS = [_line for _line in fh.readlines() if _line]
 
 # make long description from README and CONTRIBUTING
 # but move copyright notice to the end
-LONG_DESCRIPTION, COPYRIGHT = README.split("## Copyright Notice")
-LONG_DESCRIPTION += "\n" + CONTRIBUTING
-LONG_DESCRIPTION += "\n" + "## Copyright Notice" + COPYRIGHT
-
-# write metadata to a file that will be imported by becquerel
-with open("becquerel/__metadata__.py", "w") as f:
-    print('"""Becquerel package metadata."""', file=f)
-    print("", file=f)
-    print(f'__description__ = "{DESCRIPTION}"', file=f)
-    print(f'__url__ = "{URL}"', file=f)
-    print(f'__version__ = "{VERSION}"', file=f)
-    print(f'__license__ = """{LICENSE}"""', file=f)
-    print(f'__copyright__ = """{COPYRIGHT}"""', file=f)
+LONG_DESCRIPTION = "{0}\n{2}\n## Copyright Notice\n{1}".format(
+    *README.split("## Copyright Notice"), CONTRIBUTING
```

Comment:
This line highlights that the Copyright notice is in three places now: `__metadata__.py`, `LICENSE.txt`, and `README.md`. Is there a way to make `__metadata__.py` the master version of the Copyright text and then, e.g., have code in `setup.py` that automatically populates `LICENSE.txt` and `README.md` so that they are always in sync?

Reply:
Good point! I'll give that a shot.

Reply:
Added in 914d1393cdb2ed5636cf11d83a5d77145a23f789